package.json

```diff
@@ -6,7 +6,10 @@
   "dependencies": {
     "@types/node": "^14.0.13",
     "aws-lambda-fastify": "^1.4.3",
-    "fastify": "^2.14.1",
+    "aws-sdk": "^2.709.0",
```

Comment:
`aws-sdk` can be a dev dependency, the Lambda runtime includes it by default so it doesn't need to be included in the package and so we can save a bunch of space by moving it to dev dependencies.